UPGRADE-6.7.md

```diff
@@ -416,74 +539,40 @@ ui.componentSection.add({
 
 Additionally, the associated data sets `sw-dashboard-detail__todayOrderData` and `sw-dashboard-detail__statisticDateRanges` were removed.
 In both cases, use the Admin API instead.
-## Direct debit default payment: State change removed
-* The default payment method "Direct debit" will no longer automatically change the order state to "in progress". Use the flow builder instead, if you want the same behavior.
-
-## Introduced in 6.6.4.0
-## Removal of Storefront `sw-skin-alert` SCSS mixin
-The mixin `sw-skin-alert` will be removed in v6.7.0. Instead of styling the alert manually with CSS selectors and the custom mixin `sw-skin-alert`,
-we modify the appearance inside the `alert-*` modifier classes directly with the Bootstrap CSS variables like it is documented: https://getbootstrap.com/docs/5.3/components/alerts/#sass-loops
-
-Before:
-```scss
-@each $color, $value in $theme-colors {
-  .alert-#{$color} {
-    @include sw-skin-alert($value, $white);
-  }
-}
-```
-
-After:
-```scss
-@each $state, $value in $theme-colors {
-  .alert-#{$state} {
-    --#{$prefix}alert-border-color: #{$value};
-    --#{$prefix}alert-bg: #{$white};
-    --#{$prefix}alert-color: #{$body-color};
-  }
-}
-```
-
-## Removal of Storefront alert class `alert-has-icon` styling
-When rendering an alert using the include template `Resources/views/storefront/utilities/alert.html.twig`, the class `alert-has-icon` will be removed. Helper classes `d-flex align-items-center` will be used instead.
-
-```diff
-- <div class="alert alert-info alert-has-icon">
-+ <div class="alert alert-info d-flex align-items-center">
-    {% sw_icon 'info' %}
-    <div class="alert-content-container">
-        An important info
-    </div>
-</div>
-```
 
-## Removal of Storefront alert inner container `alert-content`
-As of v6.7.0, the superfluous inner container `alert-content` will be removed to have lesser elements and be more aligned with Bootstraps alert structure.
-When rendering an alert using the include template `Resources/views/storefront/utilities/alert.html.twig`, the inner container `alert-content` will no longer be present in the HTML output.
-
-The general usage of `Resources/views/storefront/utilities/alert.html.twig` and all include parameters remain the same.
-
-Before:
-```html
-<div role="alert" class="alert alert-info d-flex align-items-center">
-    <span class="icon icon-info"><svg></svg></span>                                                    
-    <div class="alert-content-container">
-        <div class="alert-content">                                                    
-            Your shopping cart is empty.
-        </div>                
-    </div>
-</div>
-```
+## Replace `isEmailUsed` with `isEmailAlreadyInUse`:
+Replace `isEmailUsed` with `isEmailAlreadyInUse` in `sw-users-permission-user-detail`.
+
+## Component replacement with Meteor Component Library
```

Comment:
I think here would be interesting how I can build a plugin that works in two major versions

Reply:
It is not possible to build a adminstration plugin which supports two major versions